<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notepad</class>
 <widget class="QMainWindow" name="Notepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notepad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
    <addaction name="actionFont"/>
    <addaction name="actionUnderline"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionUndo"/>
   <addaction name="separator"/>
   <addaction name="actionFont"/>
   <addaction name="actionUnderline"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="NotePad.qrc">
     <normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/new.png</normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New </string>
   </property>
   <property name="toolTip">
    <string>New text document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="NotePad.qrc">
     <normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/open.png</normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="NotePad.qrc">
     <normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/save.png</normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="NotePad.qrc">
     <normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/copy.png</normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="NotePad.qrc">
     <normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/underline.png</normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
   <property name="toolTip">
    <string>Underline</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="NotePad.qrc">
     <normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/info.png</normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About Notepad</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="NotePad.qrc">
     <normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/exit.png</normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit notepad</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="NotePad.qrc">
     <normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/font.png</normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/font.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
   <property name="toolTip">
    <string>Font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="NotePad.qrc">
     <normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/edit_undo.png</normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/edit_undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="NotePad.qrc">
     <normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/save_as.png</normaloff>:/D:/Qt/Examples/Qt-6.2.4/widgets/tutorials/notepad/images/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="toolTip">
    <string>Save file as</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="NotePad.qrc"/>
 </resources>
 <connections/>
</ui>
